cmp/LoadMask.js

```diff
@@ -44,7 +42,7 @@ export class LoadMask extends Component {
                 style: {backgroundColor: this.BACKGROUND}
             },
             usePortal: !inline,
-            item: inline ? this.getInlineChild() : this.getViewportChild()
+            item: inline !== false ? this.getInlineChild() : this.getViewportChild()
```

Comment:
like this way of doing the defaulting on line 45, but you are using the old way on line 44!  Seems like you need a single local variable here for this!